src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -53,6 +53,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _InitializeProfilesList();
 
         _colorSchemesNavState = winrt::make<ColorSchemesPageNavigationState>(_settingsClone);
+        _lastProfilesNavState = winrt::make<ProfilePageNavigationState>();

```

Comment:
I don't think this change is needed. This member is initialized in `_Navigate`.

Reply:
Good point, removed